src/java.desktop/share/native/liblcms/cmstypes.c

```diff
@@ -3665,7 +3665,7 @@ country varies for each element:
 
 // Auxiliary, read an string specified as count + string
 static
-cmsBool  ReadCountAndSting(struct _cms_typehandler_struct* self, cmsIOHANDLER* io, cmsMLU* mlu, cmsUInt32Number* SizeOfTag, const char* Section)
+cmsBool  ReadCountAndString(struct _cms_typehandler_struct* self, cmsIOHANDLER* io, cmsMLU* mlu, cmsUInt32Number* SizeOfTag, const char* Section)
```

Comment:
As @kevinrushforth mentioned, we usually don't modify the source from external libraries. You should report and fix the problem upstream.

Reply:
Filed https://github.com/mm2/Little-CMS/pull/313 for this